internal/workflow/local_activities.go

```diff
@@ -60,12 +60,8 @@ func updatePackageLocalActivity(ctx context.Context, logger logr.Logger, colsvc
 	return nil
 }
 
-type checkDuplicateLocalActivityParams struct {
-	Name string
-}
-
-func checkDuplicateLocalActivity(ctx context.Context, logger logr.Logger, colsvc collection.Service, params *checkDuplicateLocalActivityParams) (bool, error) {
-	return colsvc.CheckDuplicate(ctx, params.Name)
+func checkDuplicatePackageLocalActivity(ctx context.Context, logger logr.Logger, colsvc collection.Service, collection_name string) (bool, error) {
```

Comment:
The convention in Go is to use `camelCase`, e.g. `collectionName` but since here it's obvious I'd just use `name string`.

Reply:
yep, force of habit